src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +21,14 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        OptionalInt result = numbers.stream()
+                .flatMap(line -> Arrays.stream(line.split(LINE_SEPARATOR)))
+                .mapToInt(Integer::parseInt)
+                .filter(number -> number % 2 == 0)
+                .min();
+
+        return result.orElseThrow(() -> new RuntimeException("Can't get min value from list: "
+                + numbers));
```

Comment:
It is better not to divide this logic
```suggestion
        return numbers.stream()
                .flatMap(line -> Arrays.stream(line.split(LINE_SEPARATOR)))
                .mapToInt(Integer::parseInt)
                .filter(number -> number % 2 == 0)
                .min()
                .orElseThrow(() -> new RuntimeException("Can't get min value from list: " + numbers));
```